app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -25,6 +25,8 @@ def refresher_options
 
       private
 
+      # FIXME: The intention was to make this method private but tests or code called
+      # from the tests are expecting it to be public.
```

Comment:
I"m ok with dropping this comment and moving this method before the private keyword in a followup PR.